xline/src/server/command.rs

```diff
@@ -89,7 +89,7 @@ impl KeyRange {
     #[inline]
     pub fn new_one_key(key: impl Into<Vec<u8>>) -> Self {
         let key_vec = key.into();
-        assert!(key_vec.as_slice() != UNBOUNDED);
+        assert!(key_vec.as_slice() != UNBOUNDED, "key cannot be UNBOUNDED");
```

Comment:
This diagnostic message doesn't provide any useful information.  When the assertion fails, we are more concerned about what type this key is and why it is of that type.

Reply:
done